src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/DataSetMeta.cs

```diff
@@ -28,6 +28,21 @@ public class DataSetMeta : ICreatedUpdatedTimestamps<DateTimeOffset, DateTimeOff
 
     public DateTimeOffset? Updated { get; set; }
 
+    public DataSetVersionMetaSummary ToSummary()
+    {
+        return new DataSetVersionMetaSummary
+        {
+            Filters = Filters.Select(f => f.Label).ToList(),
+            Indicators = Indicators.Select(i => i.Label).ToList(),
+            GeographicLevels = GeographicLevels,
+            TimePeriodRange = new TimePeriodRange
+            {
+                Start = TimePeriods.First(),
+                End = TimePeriods.Last()
```

Comment:
Oooo is this risky? Might not be guaranteed

Reply:
Discussed in person, but essentially the assumption is that time periods should be ordered by the code that saves the `DataSetMeta` into the database.

Ordering around time periods is also quite a tricky topic so it's better being dealt outside of this mapping function anyway.